Makefile

```diff
@@ -67,6 +67,7 @@ verify:
 	hack/verify-profile-manifests.sh
 	hack/verify-generated-bindata.sh
 	hack/verify-deps.sh
+	hack/verify-e2e-test-all-presence.sh
```

Comment:
The way the commits are ordered, this commit would cause tests to fail, which would break `git bisect`, wouldn't it?

Reply:
Likely. But there are many commits now...

Reply:
I moved this addition to the last commit: https://github.com/openshift/cluster-ingress-operator/pull/756/commits